### PR TITLE
Fix code samples in the README.md

### PR DESCRIPTION
**Pull request by Biruntha**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,46 +39,43 @@ facebook:FacebookConfiguration facebookConfig = {
...
 ```
 
-The response from `createPost` is a `Post` object if the request was successful or a `error` on failure.
+The response from `createPost` is a `Post` object if the request is successful or an `error` if unsuccessful.
 ```ballerina
 if (response is facebook:Post) {
-   //If successful, returns the Post object.
-   response = response;
-   io:println(fbRes);
+   // If successful, print Post details.
```
Comment (NatashaWso2) [inline README.md]:
```suggestion
   // If successful, print the Post details.
```

File: facebook/Module.md [hunk 2]
```diff
@@ -69,159 +69,158 @@ facebook:Client facebookclient = new(facebookConfig);
...
 ```
 
-The response from `createPost` is a `Post` object if the request was successful or a `error` on failure. .
+The response from `createPost` is a `Post` object if the request is successful or an `error` if unsuccessful.
 ```ballerina
 if (response is facebook:Post) {
-   //If successful, returns the Post object.
-   response = response;
-   io:println(fbRes);
+   // If successful, print Post details.
```
Comment (NatashaWso2) [inline facebook/Module.md]:
```suggestion
   // If successful, print the Post details.
```

File: README.md [hunk 3]
```diff
@@ -39,46 +39,43 @@ facebook:FacebookConfiguration facebookConfig = {
...
-    io:println(p);
+    io:println("Post Details: ", response);
 } else {
-    io:println(response);
+    io:println("Error: ", response);
 }
 ```
 
-The `deletePost` function deletes the post specified by the ID. The `postId` represents the ID of the post to be deleted. It returns the `True` object on success and `error` on failure.
+The `deletePost` function deletes the post specified by the ID. The `postId` represents the ID of the post to be deleted. It returns the `True` object on success or an `error` if unsuccessful.
```
Comment (MaryamZi) [inline README.md]:
```suggestion
The `deletePost` function deletes the post specified by the ID. The `postId` represents the ID of the post to be deleted. It returns deletion status on success or an `error` if an error occurred. 
```

Comment (MaryamZi) [reply README.md]: Same for the Module.md too.

File: facebook/Module.md [hunk 4]
```diff
@@ -69,159 +69,158 @@ facebook:Client facebookclient = new(facebookConfig);
...
-    io:println(b);
+    io:println("Status: ", response);
 } else {
-    io:println(response);
+    io:println("Error: ", response);
 }
 ```
 
-The `getFriendListDetails` function used to get the User's friends who have installed the app making the query. The `userId` represents the ID of the user. It returns the `FriendList` object on success and `error` on failure.
+The `getFriendListDetails` function used to get the User's friends who have installed the app making the query. The `userId` represents the ID of the user. It returns the `FriendList` object on success or an `error` if unsuccessful.
```
Comment (MaryamZi) [inline facebook/Module.md]:
```suggestion
The `getFriendListDetails` remote function is used to get the user's friends who have installed the app making the query. The `userId` represents the ID of the user. It returns a `FriendList` object on success or an `error` if unsuccessful.
```

File: facebook/Module.md [hunk 5]
```diff
@@ -69,159 +69,158 @@ facebook:Client facebookclient = new(facebookConfig);
...
-    io:println(friendList); 
+    io:println("Friend List: ", response);
 } else {
-     io:println(response);
+     io:println("Error: ", response);
 }
 ```
 
-The `getPageAccessTokens` function used to get the page access tokens. The `userId` represents the ID of the user. It returns the `AccessTokens` object on success and `error` on failure.
+The `getPageAccessTokens` function used to get the page access tokens. The `userId` represents the ID of the user. It returns the `AccessTokens` object on success or an `error` if unsuccessful.
```
Comment (MaryamZi) [inline facebook/Module.md]:
```suggestion
The `getPageAccessTokens` remote function us used to get page access tokens. The `userId` represents the ID of the user. It returns an `AccessTokens` object on success or an `error` if unsuccessful.
```

File: facebook/Module.md [hunk 6]
```diff
@@ -69,159 +69,158 @@ facebook:Client facebookclient = new(facebookConfig);
...
-    io:println(response);
+    io:println("Error: ", response);
 }
 ```
 
 ##### Example
 
 ```
 import ballerina/io;
+import ballerina/http;
```
Comment (MaryamZi) [inline facebook/Module.md]: Shall we order the imports alphabetically?

File: facebook/Module.md [hunk 7]
```diff
@@ -69,159 +69,158 @@ facebook:Client facebookclient = new(facebookConfig);
...
-    string pageAccessToken = callMethodsWithUserToken(userAccessToken);
-    callMethodsWithPageToken(pageAccessToken);
-}
-
 facebook:FacebookConfiguration facebookConfig = {
     clientConfig:{
         auth:{
             scheme: http:OAUTH2,
-            accessToken:accessToken
+            accessToken: getUserAccessToken()
```
Comment (MaryamZi) [inline facebook/Module.md]: Do we need a function? We can directly say `config:getAsString("ACCESS_TOKEN");` right?

File: facebook/Module.md [hunk 8]
```diff
@@ -67,161 +67,156 @@ facebook:FacebookConfiguration facebookConfig = {
...
         }
     }
 };
 
 facebook:Client facebookclient = new(facebookConfig);
 
-function callMethodsWithUserToken(string userAccessToken) returns string {
-    
+string pageAccessToken = "";
+public function main(string... args) {
```
Comment (Shan1024) [inline facebook/Module.md]: Better to add a newline between the global variable and the main function.

File: facebook/Module.md [hunk 9]
```diff
@@ -67,161 +67,156 @@ facebook:FacebookConfiguration facebookConfig = {
 facebook:Client facebookclient = new(facebookConfig);
```
Comment (Shan1024) [inline facebook/Module.md]: Better to rename this `facebookclient` to `facebookClient` (including all the usages) as well.

File: facebook/Module.md [hunk 10]
```diff
@@ -67,161 +67,156 @@ facebook:FacebookConfiguration facebookConfig = {
...
     clientConfig:{
         auth:{
             scheme: http:OAUTH2,
-            accessToken:accessToken
+            accessToken: config:getAsString("ACCESS_TOKEN")
         }
     }
 };
 
 facebook:Client facebookclient = new(facebookConfig);
```
Comment (Shan1024) [inline facebook/Module.md]: Better to rename this `facebookclient` to `facebookClient` (including all the usages) as well.

File: facebook/Module.md [hunk 11]
```diff
@@ -67,161 +67,156 @@ facebook:FacebookConfiguration facebookConfig = {
...
+    clientConfig:{
+        auth:{
+            scheme: http:OAUTH2,
+            accessToken: getpageToken()
         }
-    };
+    }
+};
+
+facebook:Client facebookPageclient = new(facebookPageConfig);
```
Comment (Shan1024) [inline facebook/Module.md]: Better to rename `facebookPageclient` -> `facebookPageClient` including all the usages.

File: facebook/Module.md [hunk 12]
```diff
@@ -67,161 +67,156 @@ facebook:FacebookConfiguration facebookConfig = {
...
+
+function getpageToken() returns string {
+    return pageAccessToken;
+}
+
+function callMethodsWithPageToken() {
     io:println("-----------------Calling to create fb post------------------");
-    var createPostResponse = client->createPost("me","testBalMeassage","","");
-    string postId;
+    var createPostResponse = facebookPageclient->createPost("me","testBalMeassage","","");
```
Comment (Shan1024) [inline facebook/Module.md]:
```suggestion
    var createPostResponse = facebookPageclient->createPost("me", "testBalMeassage", "", "");
```

File: facebook/Module.md [hunk 13]
```diff
@@ -64,164 +64,160 @@ facebook:FacebookConfiguration facebookConfig = {
...
+   // If unsuccessful, print the error returned.
+   io:println("Error: ", response);
 }
 ```
 
-The `retrievePost` function retrieves the post specified by the ID. The `postId` represents the ID of the post to be retrieved. It returns the `Post` object on success and `error` on failure.
+The `retrievePost` remote function retrieves the post specified by the ID. The `postId` represents the ID of the post to be retrieved. It returns the `Post` object on success or an `error` if unsuccessful.
 ```ballerina
-var response = facebookEP.retrievePost(postId);
+var response = facebookClient.retrievePost(postId);
```
Comment (MaryamZi) [inline facebook/Module.md]: This needs to be `->` right? Please check other places too!
```suggestion
var response = facebookClient->retrievePost(postId);
```

File: facebook/Module.md [hunk 14]
```diff
@@ -64,164 +64,160 @@ facebook:FacebookConfiguration facebookConfig = {
...
-    io:println(response);
+    io:println("Error: ", response);
 }
 ```
 
-The `deletePost` function deletes the post specified by the ID. The `postId` represents the ID of the post to be deleted. It returns the `True` object on success and `error` on failure.
+The `deletePost` remote function deletes the post specified by the ID. The `postId` represents the ID of the post to be deleted. It returns deletion status on success or an `error` if an error occurred.
 ```ballerina
-var response = facebookEP.deletePost(postId);
+var response = facebookClient.deletePost(postId);
```
Comment (MaryamZi) [inline facebook/Module.md]:
```suggestion
var response = facebookClient->deletePost(postId);
```

File: facebook/Module.md [hunk 15]
```diff
@@ -64,164 +64,160 @@ facebook:FacebookConfiguration facebookConfig = {
...
-    io:println(response);
+    io:println("Error: ", response);
 }
 ```
 
-The `getFriendListDetails` function used to get the User's friends who have installed the app making the query. The `userId` represents the ID of the user. It returns the `FriendList` object on success and `error` on failure.
+The `getFriendListDetails` remote function is used to get the user's friends who have installed the app making the query. The `userId` represents the ID of the user. It returns a `FriendList` object on success or an `error` if unsuccessful.
 ```ballerina
-var response = facebookEP.getFriendListDetails(userId);
+var response = facebookClient.getFriendListDetails(userId);
```
Comment (MaryamZi) [inline facebook/Module.md]:
```suggestion
var response = facebookClient->getFriendListDetails(userId);
```

**Pull request merged**